fem/quadinterpolator.hpp

```diff
@@ -143,6 +148,8 @@ class QuadratureInterpolator
        @a e_vec at quadrature points. */
    void PhysDerivatives(const Vector &e_vec, Vector &q_der) const;
 
+   void AbsPhysDerivatives(const Vector &e_vec, Vector &q_der) const;
```

Comment:
Doxygen comment needed here

Reply:
Note: this is one of the things I recommended to be removed.